CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 #   - WITH_LUAROCKS : Install required luarocks in the macOS app (requires luarocks)
 
 #   - ENABLE_UNIT_TESTS : Enable Unit Testing Target (requires Catch2) (no)
-#   - BUILD_ANIMVIEWER : Generate AnimViewer build files (no)
+#   - BUILD_ANIMVIEWER or BUILD_ANIMVIEW : Generate AnimViewer build files (no)
```

Comment:
In my opinion, remove `BUILD_ANIMVIEWER` at least from the UI, and possibly delete it entirely. (The program is known as `animview`, see also line 134, having a different name can only be confusing.)

The number of people building from source is small, and the subset that doesn't build it from scratch after update is even smaller.